index.js

```diff
@@ -160,7 +164,8 @@ function findSiblingAncestors(startNodeAndParents, endNodeAndParents) {
     if (
       endParent.type !== "Program" &&
       endParent.type !== "File" &&
-      util.locStart(endParent) >= util.locStart(startNodeAndParents.node)
+      util.locStart(endParent, opts.locStart) >=
```

Comment:
Shouldn't this just be `opts.locStart(endParent) `?

Reply:
Definitely! fixed.